src/test/java/seedu/address/testutil/TypicalRooms.java

```diff
@@ -1,62 +1,109 @@
 package seedu.address.testutil;
 
-import seedu.address.model.room.DoubleRoom;
+import seedu.address.model.room.Capacity;
 import seedu.address.model.room.Room;
-import seedu.address.model.room.RoomNumber;
-import seedu.address.model.room.SingleRoom;
-import seedu.address.model.room.SuiteRoom;
 import seedu.address.model.room.UniqueRoomList;
 
 /**
  * A utility class containing a list of {@code Room} objects to be used in tests.
  */
 public class TypicalRooms {
 
-    private static final int CAPACITY_SINGLE_ROOM = SingleRoom.getCapacityValue();
-    private static final int CAPACITY_DOUBLE_ROOM = DoubleRoom.getCapacityValue();
-    private static final int CAPACITY_SUITE_ROOM = SuiteRoom.getCapacityValue();
-
     public static final Room SINGLE_001 = new RoomBuilder()
```

Comment:
Might want to rename the room to specify the default bookings included.
Would be complicated for the rooms with multiple bookings included.

Reply:
I thought of this when naming these constants. Reason for not spelling out the bookings in each room is as you say, complicated for multiple bookings. That is why I documented which room has what bookings in the docs of getTypicalRoomsWithBookings() method below, and the TypicalBookings class for further reference if you need. You should refer to it.